tests/basic.feature

```diff
@@ -32,9 +32,9 @@ Feature: Testing Sphinx Writer
         Test the additional options for a scenario
 
         Given step text
-        '''
-        Lorum ipsum solor sit amet.
-        '''
+          '''
+          Lorum ipsum solor sit amet.
+          '''
```

Comment:
Hmmm, i would not have expected to see an indentation change here...
(or in the various places below...)